Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submissions-grid/SubmissionsGrid.module.scss

```diff
@@ -10,3 +11,36 @@
   justify-content: center;
   margin-top: 20px;
 }
+
+.submissionsGrid {
+  color: colors.$white-color;
+  margin-top: 20px;
+  table-layout: auto;
+  width: 100%;
+
+  td {
+    overflow: hidden;
+    padding: 15px 10px;
+    text-align: left;
+    text-overflow: ellipsis;
+    vertical-align: middle;
+
+    .tdRight {
+      text-align: right;
+    }
+  }
+
+  .submissionsGridHeader {
+    @extend %font-family-roboto;
+  }
+
+  .lightSubmissionsGridHeader {
+    background: colors.$wrappers-border-color;
+    box-shadow: shadows.$dp-shadow-1;
+  }
+
+  .darkSubmissionsGridHeader {
+    background: #1c1c17;
```

Comment:
dont we have a variable in colors file for this?

Reply:
No :/ This is pretty much black but i got this color from figma